codeforboston/maple
# Update About Section in Header Menu (Issue #1547)

# Summary

Finished all items on issue #1547. Changed arrangement of About section in Navbar. Made the Why Use Maple into tabs instead of three separate pages.

# Checklist

- [x] Condense three separate Why Use Maple pages into three tabs on a single page.
- [x] Remove “Privacy Policy & Code of Conduct” from the About section of the menu
- [x] Re-order the subpages under About in following order: Our Mission & Goals, Our Team, How to Support MAPLE, FAQ

# Screenshots

![image_2024-06-04_170437136](https://github.com/codeforboston/maple/assets/94672626/a924a72d-865f-4d28-b0f6-085bfb36e6dd)
![MAPLE_about_changes](https://github.com/codeforboston/maple/assets/94672626/11eb4f59-09b1-43c8-95ba-48fabd657ee5)


# Known issues

No known issues

# Steps to test/reproduce

_For each feature or bug fix, create a step by step list for how a reviewer can test it out. E.g.:_

1. Go to the home page
2. Click on dropdown menu and click About to see one change, or any of the For's under Why Use Maple to see tabs.
3. See that changes made are the changes desired
